app/models/user.rb

```diff
@@ -1,4 +1,11 @@
 class User < ApplicationRecord
   has_and_belongs_to_many :organisations
   validates_presence_of :name, :email
+
+  enum role: {
+    administrator: "administrator",
+    delivery_partner: "delivery_partner",
+  }
+
+  attribute :role, :string, default: "delivery_partner"
```

Comment:
How does this decision impact adding multiple roles to a user in future?

Reply:
Particularly because we expect the total number of users not to be huge, I think it should be straightforward enough to migrate this over to a many-to-many association with a join table if needed in the future.